starter/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/pborman/uuid"
```

Comment:
FWIW, we regret using this one over https://pkg.go.dev/github.com/google/uuid which has become more standard in Go projects. But this doesn't matter much.